doc/user_guide/user_guide.md

```diff
@@ -320,6 +325,13 @@ SELECT TE_TOKEN_CLASSIFICATION_UDF(
 The inference results are presented with _START_POS_ indicating the starting index of the token, 
 _END_POS_ indicating the ending index of the token, _WORD_ indicating the token, predicted _ENTITY_, and 
 confidence _SCORE_ columns, combined with the inputs used when calling this UDF.
+For example:
+
+| BUCKETFS_CONN | SUB_DIR | MODEL_NAME | TEXT_DATA | AGGREGATION_STRATEGY | START_POS | END_POS | WORD | ENTITY | SCORE |
+| ------------- | ------- | ---------- |-----------|----------------------|-----------|---------|------|--------|-------|
+| conn_name     | dir/    | model_name | text      | simple               | 0         | 4       | text | noun   | 0.75  |
```

Comment:
are start pos and end pos, character position or token positions

Reply:
it is character position